From: Tobit Flatscher <tobit@intrinsic.ai>
Subject: [PATCH] Remove FANUC hardware interface version check

The driver is currently released for ROS 2 Jazzy only while the ICON HWM is
targeting Kilted. Remove the distribution check in CMakeLists.txt to allow
building on other ROS 2 distributions.
---
 fanuc_hardware_interface/CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/fanuc_hardware_interface/CMakeLists.txt b/fanuc_hardware_interface/CMakeLists.txt
--- a/fanuc_hardware_interface/CMakeLists.txt
+++ b/fanuc_hardware_interface/CMakeLists.txt
@@ -13,12 +13,6 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
 endif()
 set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)

-if("$ENV{ROS_DISTRO}" STREQUAL "humble")
-    message(FATAL_ERROR "This branch is targeting Jazzy Jalisco. Please use the `humble` branch for Humble Hawksbill. ")
-elseif(NOT "$ENV{ROS_DISTRO}" STREQUAL "jazzy")
-    message(FATAL_ERROR "Unknown distribution: $ENV{ROS_DISTRO}")
-endif()
-
 find_package(ament_cmake REQUIRED)
 find_package(hardware_interface REQUIRED)
 find_package(pluginlib REQUIRED)
